srv/src/main/resources/application.yml

```diff
@@ -5,7 +5,11 @@ management:
                 include: "*"
 
 cds:
-    sql.hana.optimizationMode: hex
+    sql.hana:
+      optimizationMode: hex
+      search:
+        fuzzy: true
+        fuzzinessThreshold: 0.9
```

Comment:
I did not have a deeper look at fuzzy search support, yet. But is fuzzy search then active for all OData search operations?

Reply:
Yes, this globally enables fuzzy for all search queries (CDS QL and OData) on HANA Cloud